Providers/Scripts/2.4x-2.5x/Scripts/nxOMSGenerateInventoryMof.py

```diff
@@ -135,6 +135,12 @@ def GenerateInventoyMOFContents(FeatureName, Instances, RunIntervalInSeconds, Ta
 def GenerateInventoyConfContents(FeatureName, Instances, RunIntervalInSeconds, Tag, Format, FilterType, Configuration):
     mof_file_path = inventoryMof_path + FeatureName + '.mof'
     conf_file_path = inventoryMof_path + FeatureName + '.conf'
+
+    configurations = ''
+    if Configuration is not None:
+        for configuration in Configuration:
+            configurations += '  ' + configuration + '\n'
```

Comment:
This is for indentation of the generated conf file. This code generates conf file, which needs this indentation